readthedocs/projects/tasks.py

```diff
@@ -317,6 +317,8 @@ def get_rtd_env_vars(self):
         env = {
             'READTHEDOCS': 'True',
             'READTHEDOCS_VERSION': self.version.slug,
+            'READTHEDOCS_VERSION_TYPE': self.version.type,
+            'READTHEDOCS_VERSION_NAME': self.version.explicit_name,
```

Comment:
```suggestion
            'READTHEDOCS_VERSION_NAME': self.version.verbose_name,
```

My thinking here was that users could get the original name and add any other information based on the version_type.